docs/development/affiliated-packages.rst

```diff
@@ -269,6 +269,12 @@ files manually`_ section since this explains what many of the files do.
       or just delete it.
    #. Exit out of your text editor.
 
+#. Move the main source directory to reflect the name of your package.
+   To tell your DVCS about this move, you should use it, and not ``mv``
```

Comment:
@pllim - I thought "you should use it" has a typo "it" -> "git", but the "it" can also refer to "your DVCS", so forget my comment, sorry.